sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/samples/java/com/azure/messaging/eventhubs/checkpointstore/blob/EventProcessorWithCustomStorageVersion.java

```diff
@@ -12,10 +12,17 @@
 import java.util.concurrent.CountDownLatch;
 
 /**
- * This sample shows how to setup an Event Processor that uses an older version of blob storage service as
- * checkpoint store.
+ * This sample shows how to setup an Event Processor that uses a different version of blob storage service that is
+ * not directly supported by Storage Blob SDK as checkpoint store.
+ *
+ * The following sample can be used if the environment you are targeting supports a different version of Storage Blob
+ * SDK than those typically available on Azure. For example, if you are running Event Hubs on an Azure Stack Hub version
+ * 2002, the highest available version for the Storage service is version 2017-11-09. In this case, you will need to use
+ * the following code to change the Storage service API version to 2017-11-09. For more information on the Azure Storage
+ * service versions supported on Azure Stack Hub, please refer to
+ * <a href=docs.microsoft.com/en-us/azure-stack/user/azure-stack-acs-differences>Azure Stack Hub Documentation</a>
```

Comment:
nit: remove en-us from URL. It'll redirect to the correct locale.

Reply:
Good catch, updated!